lib/utils/app_util.dart

```diff
@@ -33,3 +37,54 @@ class MessageInCenterWidget extends StatelessWidget {
     );
   }
 }
+
+bool confirmAlert(
+  BuildContext context, {
+  Function()? onConfirm,
+  Function()? onCancel,
+  String title = 'CONFIRMATION',
```

Comment:
We should this a required parameter instead of giving it default.

Reply:
`title` and `desc` are now required.